actioncable/test/channel/base_test.rb

```diff
@@ -145,4 +145,8 @@ def rm_rf
     assert_equal expected, @connection.last_transmission
   end
 
+  test "actions available on Channel" do
+    available_actions = ["room", "last_action", "subscribed", "unsubscribed", "toggle_subscribed", "leave", "speak", "subscribed?", "get_latest", "chatters", "topic"].to_set
```

Comment:
I will change it in next PR taking stab at https://github.com/rails/rails/pull/22664/files#r48028620